ee/ucp/admin/install/install-on-azure.md

```diff
@@ -4,81 +4,78 @@ description: Learn how to install Docker Universal Control Plane in a Microsoft
-`Orchestrator` tag. Note the value for this tag is the Kubernetes version number
-in the format `Orchestrator=Kubernetes:x.y.z`. This value may change in each 
-UCP release. To find the relevant version please see the UCP 
-[Release Notes](../../release-notes). For example for UCP 3.1.0 the tag 
-would be `Orchestrator=Kubernetes:1.11.2`. 
-- The Azure Virtual Machine Object Name needs to match the Azure Virtual Machine 
-Computer Name and the Node Operating System's Hostname. Note this applies to the 
-FQDN of the host including domain names. 
-- An Azure Service Principal with `Contributor` access to the Azure Resource 
-Group hosting the UCP Nodes. Note, if using a separate networking Resource 
-Group the same Service Principal will need `Network Contributor` access to this 
-Resource Group.
+- All UCP worker and manager nodes need to be attached to the same Azure
+  Subnet.
+- All UCP workers and managers need to have the `Orchestrator` tag in Azure with
+  the Kubernetes version as the value, following this format:
+  `Orchestrator=Kubernetes:x.y.z`. This value may change in each UCP release. To
+  find the relevant version, see the UCP [Release
+  Notes](/ee/ucp/release-notes). For example, the tag for UCP `3.1.0` would be
+  `Orchestrator=Kubernetes:1.11.2`. 
+- The Azure Virtual Machine Object Name needs to match the Azure Virtual Machine
+  Computer Name and the Node Operating System's Hostname which is the FQDN of
+  the host, including domain names. Note that this requires all characters to be in lowercase.
+- An Azure Service Principal with `Contributor` access to the Azure Resource
+  Group hosting the UCP Nodes. This Service principal will be used by Kubernetes
+  to communicate with the Azure API. The Service Principal ID and Secret Key are
+  needed as part of the UCP prerequisites. If you are using a separate Resource
+  Group for the networking components, the same Service Principal will need
+  `Network Contributor` access to this Resource Group.
 
 UCP requires the following information for the installation:
 
 - `subscriptionId` - The Azure Subscription ID in which the UCP 
 objects are being deployed. 
 - `tenantId` - The Azure Active Directory Tenant ID in which the UCP 
 objects are being deployed. 
-- `aadClientId` - The Azure Service Principal ID
-- `aadClientSecret` - The Azure Service Principal Secret Key
+- `aadClientId` - The Azure Service Principal ID.
+- `aadClientSecret` - The Azure Service Principal Secret Key.
 
 ### Azure Configuration File
 
-For Docker UCP to integrate into Microsoft Azure, you need to place an Azure 
-configuration file within each UCP node in your cluster, at 
-`/etc/kubernetes/azure.json`. The `azure.json` file needs 0644 permissions.
+For Docker UCP to integrate with Microsoft Azure,each UCP node in your cluster needs an Azure 
+configuration file, `azure.json`. Place the file within `/etc/kubernetes`, and give it 0644 
```

Comment:
@ollypom can we clarify why the configuration file needs 0644 permissions?

Reply:
@bermudezmt No problem. Provided a little bit more context in the latest commit. Is that enough?

Reply:
Thank you, @ollypom . Note to myself to revisit your Slack message and add a bit more detail around `0644` file permissions.